.gitignore

```diff
@@ -62,12 +62,14 @@ install_manifest.txt
 # vim temp files
 *.swp
 *.un~
+*.*~
```

Comment:
Can we drop these changes?  This PR is big enough :).  Feel free to put them in a separate PR and I'll stamp it.